### PR TITLE
Edit and revision of overview & quick start doc 

### PR DESCRIPTION
**Pull request by SvenDowideit**
This replaces #901 

@aanand can you tell us if we should continue to maintain the old documentation meta-data?

### CODE REVIEW

**Comment by SvenDowideit** [comment]
@fredlf :)

**Comment by aanand** [comment]
Don't worry about the old docs - once Compose 1.1 is out, I'm going to maintain the Fig docs on a separate branch and deploy to GitHub Pages from there.

**Comment by fredlf** [comment]
LGTM

File: docs/index.md [hunk 1]
```diff
@@ -1,22 +1,32 @@
...
+## Overview
+
+Compose is a tool that allows you to orchestrate multiple Docker containers.
+With Compose, you can build clusters of containers which provide the resources
+(services, volumes, etc.) needed to build and run a complete distributed
+application.
+
+You can use Compose to build your app with containers hosted locally, or on a
+remote server, including cloud-based instances. Compose can also be used to
+deploy code to production.
```
Comment (bfirsh) [inline docs/index.md]: Thanks for writing this @SvenDowideit.

FYI – We're not saying that Compose can be used in production yet. I'll take a look over this next week to make sure our messaging is right.

Comment (bfirsh) [inline docs/index.md]: Oops! I see this was @fredlf. :smile: 

I know we didn't talk about this, so don't worry at all.

Comment (bfirsh) [inline docs/index.md]: @aanand @SvenDowideit @fredlf Would you mind if I could get a LGTM on each docs change? At least until the 1.1.0 release, then we can figure out a better process.

**Pull request merged**